.github/workflows/ci-rust.yml

```diff
@@ -144,6 +144,14 @@ jobs:
         run: cargo fmt --check
         timeout-minutes: 2
 
+      - name: Check restricted usage with cargo-deny
+        run: |
+          echo "::add-matcher::.github/custom-problem-matchers/cargo-deny.json"
+          # cargo-deny output the report on stderr. that output need to be redirected to stdout for the problem matcher to work.
```

Comment:
```suggestion
          # cargo-deny outputs the report on stderr and it needs to be redirected to stdout for the problem matcher to work.
```

Reply:
Why the plural form of `output` here ?

Reply:
Cause it seems that you are using "output" as a verb instead of subject: 
- cargo-deny output**s** the report on stderr
vs
- cargo-deny('s) output is reported on stderr

Anyway it was just a suggestion, the PR is :+1: